core/services/pipeline/task.bridge.go

```diff
@@ -48,11 +51,13 @@ func (t *BridgeTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inp
 		name              StringParam
 		requestData       MapParam
 		includeInputAtKey StringParam
+		cacheTTL          Uint64Param
 	)
 	err = multierr.Combine(
 		errors.Wrap(ResolveParam(&name, From(NonemptyString(t.Name))), "name"),
 		errors.Wrap(ResolveParam(&requestData, From(VarExpr(t.RequestData, vars), JSONWithVarExprs(t.RequestData, vars, false), nil)), "requestData"),
 		errors.Wrap(ResolveParam(&includeInputAtKey, From(t.IncludeInputAtKey)), "includeInputAtKey"),
+		errors.Wrap(ResolveParam(&cacheTTL, From(ValidDurationInSeconds(t.CacheTTL), t.config.BridgeCacheTTL().Seconds())), "cacheTTL"),
```

Comment:
Because we added a new task parameter, we need to update user documentation as well (separate ticket, PR): https://github.com/smartcontractkit/documentation

Reply:
Working on it